src/__tests__/spectral.test.ts

```diff
@@ -128,11 +128,14 @@ describe('spectral', () => {
     describe('and a custom resolver is provided', () => {
       test('will call the resolver with target', async () => {
         const customResolver: IResolver = {
-          resolve: jest.fn(async () => ({
-            result: {},
-            refMap: {},
-            errors: [],
-          })),
+          resolve: jest.fn(
+            async () =>
+              ({
+                result: {},
+                refMap: {},
+                errors: [],
```

Comment:
Let's avoid the type trickery here.
Add the following import
`import { DepGraph } from 'dependency-graph';` 
and then you should be able to include `graph`
```ts
errors: [],
graph: new DepGraph(),
```

Reply:
@P0lip My bad. It's now fixed.